lazy_tensor_core/test/test_lazy.py

```diff
@@ -41,6 +66,43 @@ def clone_move(t):
         torch.testing.assert_allclose(weight_copy_grad.cpu(), weight_grad.cpu())
         torch.testing.assert_allclose(inp_copy_grad.cpu(), inp_grad.cpu())
 
+class TestLazyOpInfo(TestCase):
+    @ops([op for op in op_db if op.name in LAZY_OPS_LIST and op.name not in BAD_OPS_LIST], allowed_dtypes=(torch.float,))
+    def test_dispatched_to_lazy(self, device, dtype, op):
+
+        def get_name(op):
+            l = [op.name]
+            if op.variant_test_name != '':
+                l.append(op.variant_test_name)
+            return '.'.join(l)
+
+        global FALLBACK_LIST
+        samples = op.sample_inputs("lazy", dtype, requires_grad=False)
+        sample = list(samples)[0]
+        args = [sample.input] + list(sample.args)
+        kwargs = sample.kwargs
+        ltm.mark_step()
+        ltm.wait_device_ops()
+        metrics.reset_metrics()
+
+        r = op(*args, **kwargs)
+        ltm.mark_step()
+        ltm.wait_device_ops()
+        prefix = "aten" if op.name in FALLBACK_LIST else "lazy"
+        found = f"{prefix}::{op.name}" in remove_suffixes(metrics.counter_names())
+        # check aliases
+        if not found:
+            for alias in op.aliases:
+                alias_found = f"{prefix}::{alias.name}" in remove_suffixes(metrics.counter_names())
+                found = found or alias_found
+                if found:
+                    break
+        self.assertTrue(found)
+
+# TODO: after we move to master, add Lazy as a new Device here:
```

Comment:
2 questions about the device

- would lazy mode still work with opinfo tests easily?  i think so, we'd just enforce cuda/cpu device and set the mode inside the test?
- for xla device, i think we would still test some special lazy things such as 'dispatched_to_lazy' inside our own special opinfo test, but in general do we mostly lean on existing test coverage from normal pytorch op tests, just by setting device=xla?  (I think that's already done by torch/xla today)

Reply:
> would lazy mode still work with opinfo tests easily? i think so, we'd just enforce cuda/cpu device and set the mode inside the test?

it would work for the tests that would be written the way you described. 
It would be nice if lazy mode would also be automatically tested for any other opinfo test that's device-agnostic. 

> for xla device, i think we would still test some special lazy things such as 'dispatched_to_lazy' inside our own special opinfo test, but in general do we mostly lean on existing test coverage from normal pytorch op tests, just by setting device=xla? (I think that's already done by torch/xla today)

👍